### ISSUE TITLE
Node.Cincy (07/08/2015) follow-up

### ISSUE THREAD

**Issue by fiveisprime** [nodeschool/cincinnati#6]
Turnout and participation were fantastic! There were a lot of new faces that I
hope to see here and, overall, I am incredibly happy with how things went.

These are my thoughts to get us going and, if anybody has anything else to add,
please comment and I'll keep this issue up to date with everything as our guide
to the next event.
### Things that we should keep doing

I'd love to hear more about this from the attendees but, from what I could see,
people were progressing nicely in the exercises and getting help when they
needed it.
- Energy levels and excitement were good
- Pace seems to be about what it should be
- Mentors were amazing
- Attendee to mentor ratio was great (5:1)
- "Woo"s helped keep the energy up
### Things that we should start doing

I feel really good about how things went, but we can definitely improve.
- Prepare an actual presentation for the intro
### Things that we can do differently

I need to follow my script (or maybe not be the one to present). I had a nice
script for the things I was planning to go over then totally forgot to go over
those things.
- Get the laptop hooked up to the TV way before starting
- Structure the time to work on exercises better ("work on this one for 15
  minutes with help from mentors")
- Make sure that the venue can handle as many people as what RSVP'd - we got
  lucky this time

---

Again, this was a wildly successful first session ad I'm incredibly excited to
do more of these events. Don't be shy about commenting on this issue with
anything that would make us better. :)

**Comment by tzmanics** [comment]
I agree with all of these ☝I think the attendee to mentor rate was pretty good and I felt like I was actually able to talk to/work with a good amount of people.

Although it may be hard, it would be great to have the next NS event planned so when people ask about upcoming plans we can have that info. It would also be nice to have one of us walk around and see if people want to be a member of the group there so it's easy to get in touch.

Great fun though, thanks to everyone that came!!

**Comment by anaptfox** [comment]
It was so awesome. I'm glad I was apart of this. Seeing someone's VERY FIRST "Hello World" was so heart warming :heart: 

I think we might find value in inviting people with no programming experience. I would like to see more people programming in general. The fact that they will be starting with Node.js/JavaScript is the icing on the cake.

**Comment by mattpolicastro** [comment]
Also enjoyed last night. Bump for bringing in folks who have no/little experience, but maybe have some structure for folks who are a bit more familiar with Node/JS? Talk on package/lib, new project, pair programming with unexperienced/experienced; might be good for building regular attendance as well as keeping folks involved.

**Edit:** Just to be clear, I don't think the focus should be moved away from teaching people—it was just awfully nice to see all those folks in the same room, and would love for that to continue even if it's another sort of event entirely.
